enhancements/workload-partitioning/wide-availability-workload-partitioning.md

```diff
@@ -0,0 +1,428 @@
+---
+title: wide-availability-workload-partitioning
+authors:
+  - "@eggfoobar"
+reviewers:
+  - "@jerpeter1"
+  - "@mrunalp"
+  - "@rphillips"
+  - "@browsell"
+  - "@haircommander"
+approvers:
+  - "@jerpeter1"
+  - "@mrunalp"
+api-approvers:
+  - None
+creation-date: 2022-08-03
+last-updated: 2022-08-09
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-5562
+see-also:
+  - "/enhancements/workload-partitioning"
+  - "/enhancements/node-tuning/pao-in-nto.md"
+---
+
+# Wide Availability Workload Partitioning
+
+## Summary
+
+This enhancements builds on top of the [Management Workload
+Partitioning](management-workload-partitioning.md) and the [move of PAO in
+NTO](../node-tuning/pao-in-nto.md) enhancement to provide the ability to do
+workload partitioning to our wider cluster configurations. The previous workload
+partitioning work is isolated to Single Node cluster configurations, this
+enhancement will seek to allow customers to configure workload partitioning on
+HA as well as Compact(3NC) clusters.
+
+## Motivation
+
+Customers who want us to reduce the resource consumption of management workloads
+have a fixed budget of CPU cores in mind. We want to use normal scheduling
+capabilities of kubernetes to manage the number of pods that can be placed onto
+those cores, and we want to avoid mixing management and normal workloads there.
+Expanding on the already build workload partitioning we should be able to supply
+the same functionality to HA and 3NC clusters.
+
+### User Stories
+
+As a cluster creator I want to isolate the management pods of Openshift in
+compact(3NC) and HA clusters to specific CPU sets so that I can isolate the
+platform workloads from the application workload due to the high performance and
+determinism required of my applications.
+
+### Goals
+
+- This enhancement describes an approach for configuring OpenShift clusters to
+  run with management workloads on a restricted set of CPUs.
+- Clusters built in this way should pass the same kubernetes and OpenShift
+  conformance and functional end-to-end tests as similar deployments that are
+  not isolating the management workloads.
+- We want to run different workload partitioning on masters and workers.
+- Customers will be supplied with the advice of 4 hyperthreaded cores for
```

Comment:
This will be use case specific.

Reply:
Would it be fair to say general advice is 4/2 master/worker config otherwise customer specific advice is per use case, or should we remove that line all together?

Reply:
Yes, agree.

Reply:
Sounds good, removed.